docs/Changelog.md

```diff
@@ -4670,6 +4707,43 @@ opset_import {
 <dd>Constrain input and output types to float tensors.</dd>
 </dl>
 
+### <a name="Size-1"></a>**Size-1**</a>
+
+  Takes a tensor as input and outputs an 1D int64 tensor with single element that contains the total number of elements of the input tensor.
```

Comment:
Might be good to have an example here, but this one seems more clear

Reply:
Same as above.